desktop/core/src/desktop/conf.py

```diff
@@ -1821,9 +1821,18 @@ def get_instrumentation_default():
   key='editor_autocomplete_timeout',
   type=int,
   default=30000,
-  help=_('Timeout value in ms for autocomplete of columns, tables, values etc. 0 = disabled.')
+  help=_('Deprecated')
 )
 
+DISABLE_SOURCE_AUTOCOMPLETE = Config(
+  key='disable_source_autocomplete',
+  type=coerce_bool,
+  default=False,
+  help=_('Choose whether the editor autocomplete should gather suggestions from external sources or not.')
+)
+def is_source_autocomplete_disabled():
+  return DISABLE_SOURCE_AUTOCOMPLETE.get() or EDITOR_AUTOCOMPLETE_TIMEOUT.get() == 0
```

Comment:
Curious: Why not simply remove the `EDITOR_AUTOCOMPLETE_TIMEOUT` dead config?

Reply:
IIUC, both the reasons in PR description are fulfilled now by changing to the new config (one reason is not relevant also now) and now I think we are not using this config anywhere except this method so simply removing such configs is much better? That way we only have one config or am I missing something?

Reply:
Thought it couldn't hurt keeping it around as customers might have set it to 0 in the past, so both configs will work to disable.